docs/root/api/starting_envoy.rst

```diff
@@ -0,0 +1,74 @@
+.. _api_starting_envoy:
+
+Starting Envoy
+==============
+
+---------------
+``EnvoyClient``
+---------------
+
+Starting an instance of Envoy Mobile for making requests is done by creating an ``EnvoyClient``,
+which conforms to the ``HTTPClient`` interface.
+
+To do so, create an ``EnvoyClientBuilder`` and call ``build()``:
```

Comment:
I would mention here that we have exposed some configuration options programatically. So that people are aware that that is what is happening with .addLogLevel, addStatsFlushSeconds, etc.

Reply:
Good call.